openassessment/xblock/team_mixin.py

```diff
@@ -22,6 +22,15 @@ def teams_service(self):
             return self.runtime.service(self, 'teams')
         except NoSuchServiceError:
             logger.error(u'{}: Teams service unavailable'.format(self.location))
+            raise
+
+    @cached_property
+    def teams_config_service(self):
```

Comment:
The second tiniest possible nit: should we name this (and other things below) using the full word "configuration" so we don't have to think too much about any possible differences between the service key on the LMS side and what things are named on the ORA side?

Reply:
that seems like a good idea.